client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -66,185 +68,223 @@ export const useStocktakeColumns = ({
   const t = useTranslation();
   const { getColumnPropertyAsString, getColumnProperty } = useColumnUtils();
 
-  return useColumns<StocktakeLineFragment | StocktakeSummaryItem>(
+  const packVariantsEnabled = usePackVariantsEnabled();
```

Comment:
:nit we've used the convention of the prefix `is` for boolean variables
```suggestion
  const isPackVariantsEnabled = usePackVariantsEnabled();
```

which makes me wonder if that should be `arePackVariantsEnabled` 🤔 funnily enough, this has never come up before

( it should still be `is` - conventions are conventions )